apps/dotcom/client/src/utils/consoleBranding.ts

```diff
@@ -0,0 +1,21 @@
+const TLDRAW_BRANDING_MESSAGE = `
+.*@@@@@@@@@@@@@@@@@@@@@@@@@#-                                                                                           
+%@@@@@@@@@@@@@@@@@@@@@@@@@@@@-                                                                                          
+@@@@@@@@@@@@@%%%@@@@@@@@@@@@@=     
+@@@@@@@@@@@@+:..-%@@@@@@@@@@@=     
+@@@@@@@@@@@@-    #@@@@@@@@@@@=     
+@@@@@@@@@@@@%*+*%@@@@@@@@@@@@=     
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@=      Build with tldraw at tldraw.dev
+@@@@@@@@@@@@*-::+@@@@@@@@@@@@=                
+@@@@@@@@@@@@:    *@@@@@@@@@@@=      Work with tldraw at tldraw.dev/careers
```

Comment:
If we add the protocol prefix these two links can then be clicked, feels like it's a better experience.

It does make the required console width a bit wider though, so not sure if that's the reason why it's omitted?

Reply:
Agreed, yeah it would be a much better way to display the links. Not sure, I only removed the "tldraw" ascii art bit. I'll add full link :)